ddtrace/internal/telemetry/data/payload.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict
+from typing import List
+
+from ....version import get_version
+from .dependency import Dependency
+from .dependency import create_dependency
+from .integration import Integration
+from .metrics import Series
+
+
+class Payload:
```

Comment:
This might be better as an `abc.ABC` abstract class.

Then you define `to_dict` method which raises a `NotImplementedError()`, and you could do the same for `REQUEST_TYPE`, make it a method like `request_type()` that has to be implemented by all sub-classes of this.